### ISSUE TITLE
[Sybil Report] LayerZero 

### ISSUE THREAD

**Issue by ArtemBrest** [LayerZero-Labs/sybil-report#6299]
# Reported Addresses
1) 0x5b838a90731a9d62760023e9087097aecace596c
2) 0xc94284cbcd2ca24c920ddf6da142070917a68854
3) 0x768cffd48018a9beed9e2d38edf07160c027a0ee
4) 0x5a1626175f45430322b13f8807203db8c20a9326
5) 0x9465b7579cf7e0b1b190d0d982175a07b944a311
6) 0xfb177c638fcad4f2c73d273f83274cb2e4e317ac
7) 0x84ebac00263fc6b024b72b353553da300c9bd0ea
8) 0x890bce57770a115bc3e1c0b7feae93bf3ba4a811
9) 0x5dd7618af9db6d1fd1f62f3048e6ee781ab2face
10) 0x0102639614cd5d7a2a43a321b4fbbd082c6d844d
11) 0xe44af53c4a408347060d300b2e344a85dfb8aee3
12) 0x8ce7a360873361957b0ac2321a381a5e033d58b3
13) 0x2eeb252ed003b42473da9a6540958c1b45d7405d
14) 0xece9de1a8b24b470d1d60c4745a2f154a0ef2e97
15) 0x86b95d02926d8ed6f93062b7839be86e617a05a4
16) 0x92b390e8042e0cd605bb2ded1964be5e2b102139
17) 0xd57b6a6229d803cf69f4daf5aab6abe911e34642
18) 0x638093f01c8041dc628828d83cdfd8bb22686032
19) 0x2c5ff020e1744a6d83f49e13fe2abcbaa020e528
20) 0x87208c49011fb23c05dc4f55aee66bfad26975ea

# Description
EN: My decision about wallets is due to the fact that I want more real users to receive their rightful rewards, and I do not recognize any deception when a person makes tens and hundreds of wallets. This person, an acquaintance of mine, published screenshots of the following content in some telegrams. With the goal of showing oneself above others and showing off. I believe that this should be stopped. All wallets behaved as differently as possible. There were better and worse. But a separate person.

RU: Мое решение кошельков связано с тем, что я хочу, чтобы больше реальных пользователей получили свои законные награды, и любой обман, когда человек делает десятки и сотни кошельков, я не признаю. Этот человек, мой знакомый, опубликовал в некоторых телеграммах скриншоты следующего содержания. С целью себя показать выше остальных и показать. Я считаю, что такое должно пресекаться. Все кошельки вели себя максимально разнообразно. Были лучше и хуже. Но отдельный человек.

# Detailed Methodology & Walkthrough
EN: Telegram wallet owner nickname: https://t.me/artemka_peredozz
RU: Ник владельца кошелька в телеграм: https://t.me/artemka_peredozz
EN: Name of the channel where the publications were found: https://t.me/keepcalm_play2earn/272513
RU: Название канала, где были обнаружены публикации: https://t.me/keepcalm_play2earn/272513

EN: !!!Attention, all publications may be deleted after my report. Therefore, I ask you to pay attention and check these wallets. Thank you!
RU: !!!Внимание, все публикации, после моего репорта могут быть удалены. Поэтому прошу обратить внимание и проверить данные кошельки. Спасибо!

POST: https://t.me/keepcalm_play2earn/272513/581330

![photo_2024-03-23_01-36-44](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/07fb576a-40cb-4167-95e6-4bd7dc231880)

EN: And another from the same person: https://t.me/keepcalm_play2earn/272513/582647
RU: И ещё такое от этого же человека: https://t.me/keepcalm_play2earn/272513/582647

![photo_2024-05-18_14-46-00](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/9535de22-54eb-46de-a7bf-c88d517de783)

![photo_2024-05-18_14-09-50](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/5f1e44c4-205a-453d-95dc-e0d427bb3a62)

EN: Other examples of posts of this nature:  https://t.me/keepcalm_play2earn/272513/602793
RU: Другие примеры постов такого характера: https://t.me/keepcalm_play2earn/272513/602793

![photo_2024-05-18_14-47-03](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/a9ab295a-f0f7-460d-8927-cf563e2fbe94)

# Reward Address (If Eligible)
0x1881bb7acadde4dcd070b7551e2f34ab45ef1578

**Comment by daniil45korzich** [comment]
Yes, I'm also in this chat and I know this guy. He makes over 20 accounts, I'm sure that's all you could find, bro?

**Comment by bryanfun666** [comment]
Yes, I know this guy too. He creates more than 20 accounts, I'm sure. Is that really all you could find, bro? I wanted to post these wallets here, but it’s a pity I didn’t have time.

**Comment by ArtemBrest** [comment]
> Yes, I know this guy too. He creates more than 20 accounts, I'm sure. Is that really all you could find, bro? I wanted to post these wallets here, but it’s a pity I didn’t have time.

Yes, he already admitted it himself in the chat

![photo_2024-05-18_15-16-36](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/7fd7565b-48ac-416c-a1bd-da8457b08777)
![photo_2024-05-18_15-17-07](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/5d1e0574-1080-40d9-8c12-7c4870386f30)
![photo_2024-05-18_15-18-09 (2)](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/6887f74a-02cc-43f5-982b-74531a080e4c)
![photo_2024-05-18_15-17-59](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/ca39f772-494a-4046-bf8e-da7de9b4f274)
![photo_2024-05-18_15-18-09](https://github.com/LayerZero-Labs/sybil-report/assets/44066790/0c97c738-2b7f-445e-840e-6cba977cba8c)

**Comment by feandrrrr** [comment]
Number 14 is mine wallet, I don't even know this person on screenshot!

**Comment by bryanfun666** [comment]
> Номер 14 — мой кошелек, я даже не знаю этого человека на скриншоте!

Yes, it doesn't match the photo. Now I found the wallet that is shown in the photo. You need to ask the author to replace wallet number 14 with this one
0xce70a429dd7884398875ef2d714a9fbdf6b92e97

**Comment by ArtemBrest** [comment]
> Number 14 is mine wallet, I don't even know this person on screenshot!

I corrected it to the one with the photo 0xce70a429dd7884398875ef2d714a9fbdf6b92e97

Thank you for your attention!

**Comment by ArtemBrest** [comment]
> > Номер 14 — мой кошелек, я даже не знаю этого человека на скриншоте!
> 
> Yes, it doesn't match the photo. Now I found the wallet that is shown in the photo. You need to ask the author to replace wallet number 14 with this one 0xce70a429dd7884398875ef2d714a9fbdf6b92e97

Thank you!
